### PR TITLE
Fix documentation deployment [doc only]

### PR DESCRIPTION
**Pull request by farodin91**
Todo
* fix deployment on 0.3 and 0.2 (have to be done on release branches)
* remove travis doc deployment from release branches

Signed-off-by: Jan Jansen <jan.jansen@gdata.de>

-----

Thank you for contributing to JanusGraph!

In order to streamline the review of the contribution we ask you
to ensure the following steps have been taken:

### For all changes:
- [ ] Is there an issue associated with this PR? Is it referenced in the commit message?
- [ ] Does your PR body contain #xyz where xyz is the issue number you are trying to resolve?
- [ ] Has your PR been rebased against the latest commit within the target branch (typically `master`)?
- [ ] Is your initial contribution a single, squashed commit?

### For code changes:
- [ ] Have you written and/or updated unit tests to verify your changes?
- [ ] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under [ASF 2.0](https://www.apache.org/legal/resolved.html#category-a)?
- [ ] If applicable, have you updated the LICENSE.txt file, including the main LICENSE.txt file in the root of this repository?
- [ ] If applicable, have you updated the NOTICE.txt file, including the main NOTICE.txt file found in the root of this repository?

### For documentation related changes:
- [ ] Have you ensured that format looks appropriate for the output in which it is rendered?
- [ ] If this PR is a documentation-only change, have you added a `[doc only]`
  tag to the first line of your commit message to avoid spending CPU cycles in
  Travis CI when no code, tests, or build configuration are modified?

### Note:
Please ensure that once the PR is submitted, you check Travis CI for build issues and submit an update to your PR as soon as possible.

### CODE REVIEW

File: .github/workflows/ci-docs.yml [hunk 1]
```diff
@@ -71,19 +71,21 @@ jobs:
...
       - run: curl -sfL https://raw.githubusercontent.com/traefik/structor/master/godownloader.sh | bash -s -- -b ./bin ${STRUCTOR_VERSION}
       - run: sudo ./bin/structor -o janusgraph -r janusgraph 
             --force-edit-url 
-            --rqts-url="https://raw.githubusercontent.com/janusgraph/janusgraph/master/docs/requirements.txt" 
-            --dockerfile-url="https://raw.githubusercontent.com/janusgraph/janusgraph/master/docs/docs.Dockerfile" 
-            --dockerfile-name="default-docs.Dockerfile" 
-            --menu.js-url="https://raw.githubusercontent.com/janusgraph/janusgraph/master/docs/theme/structor-menu.js.gotmpl" 
+            --rqts-url="https://raw.githubusercontent.com/JanusGraph/janusgraph/master/requirements.txt" 
+            --dockerfile-url="https://raw.githubusercontent.com/JanusGraph/janusgraph/master/docs.Dockerfile" 
+            --dockerfile-name="https://raw.githubusercontent.com/JanusGraph/janusgraph/master/docs.Dockerfile"
```
Comment (porunov) [inline .github/workflows/ci-docs.yml:81]: Just a question, `dockerfile-name` should be the same as `dockerfile-url`?

**Review by porunov — APPROVED** [review]
LGTM. Only one question below

File: .github/workflows/ci-docs.yml (hunk 1 above)
Comment (farodin91) [reply .github/workflows/ci-docs.yml:81]: Their is bug inside of structor.

Comment (porunov) [reply .github/workflows/ci-docs.yml:81]: Got it. Thanks!

**Pull request merged**